src/components/FilterTabs/Tabs.tsx

```diff
@@ -1,24 +1,27 @@
 'use client'
-import { useState } from 'react'
 import Tab from './Tab'
-import styles from './tabs.module.scss'
 import { PublishingState } from '@prisma/client'
 import { useModerationFilter } from '@/contexts/ModerationFilterContext'
 
+import styles from './tabs.module.scss'
+
 export const availableTabs: PublishingState[] = Object.values(PublishingState)
   .filter((key) => key !== 'DRAFT')
   .map((key) => PublishingState[key] as PublishingState)
 
 export default function Tabs() {
-  const { pendingStateCounter, setPublishingStateFilter } =
-    useModerationFilter()
-  const [activeTab, setActiveTab] = useState<PublishingState>(
-    PublishingState.PENDING,
-  )
+  const {
+    pendingStateCounter,
+    setPublishingStateFilter,
+    activeTab,
+    setActiveTab,
+    setEmailSearchValue,
+  } = useModerationFilter()
 
   const setModerationFilter = (tab: PublishingState) => {
-    setActiveTab(PublishingState[tab])
+    setActiveTab && setActiveTab(PublishingState[tab])
```

Comment:
**nitpick:** There should not be case when such handler is undefined. We should throw error from `useMorderationFilter` when context is not found